packages/toolpad-app/package.json

```diff
@@ -96,12 +96,14 @@
     "react": "^18.2.0",
     "react-devtools-inline": "^4.26.1",
     "react-dom": "^18.2.0",
+    "react-draggable": "^4.4.5",
```

Comment:
I think I prefer to keep it simple until we think a bit more hollistically about the UX of the properties/binding editors. Perhaps it should be a popver, like retool. Or an input in the component panel, like dynaboard? Or inline in the page, like notion? Or in a popup with a live preview? Or something else?

Reply:
Fair enough, this can be kept out of this PR